src/codegen/llvm/codegen_amdgpu.cc

```diff
@@ -136,11 +138,15 @@ class CodeGenAMDGPU : public CodeGenLLVM {
 runtime::Module BuildAMDGPU(Array<LoweredFunc> funcs, std::string target) {
   CHECK(1) << target;
   CHECK(target.length(
-) >= 5 &&
-        target.substr(0, 5) == "rocm");
-  llvm::TargetMachine* tm = GetLLVMTargetMachine(
-      " -mcpu=gfx900" +
-      target.substr(5, target.length() - 5));
+) >= 4 &&
+        target.substr(0, 4) == "rocm");
```

Comment:
I would recommend use GetLLVMTargetMachine so futher options can be passed here

Reply:
Fixed!